meta-balena-common/recipes-core/initrdscripts/files/migrate

```diff
@@ -120,6 +127,21 @@ migrate_run() {
         # shellcheck disable=SC2086
         cp -a ${kernel_images} "/tmp"
 
+        # If this is a Jetson Orin device, the flasher image contains
+        # a compressed boot blob as well as a UEFI capsule.
+        # If the QSPI is accessible, it will be written directly,
+        # otherwise a capsule update will be triggered on the next boot
+        # after provisioning
+        if [ -f "${jetson_orin_capsule}" ]; then
+            cp "${jetson_orin_capsule}" "/tmp/"
+            info "Copied ${jetson_orin_capsule}"
+        fi
+
+        if [ -f "${jetson_orin_boot_artifact}" ]; then
+            cp "${jetson_orin_boot_artifact}" "/tmp/"
+            info "Copied ${jetson_orin_boot_artifact}"
+        fi
+
```

Comment:
Again, this would need a generic hook. Otherwise, just replace the `migrate` module in the device repository with these changes but they need to be covering a generic use case to go into meta-balena.

Reply:
same as above, I dropped all changes to the migrate script here